src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Services/BlobStorageServiceTests.cs

```diff
@@ -615,7 +610,8 @@ public async Task DeleteBlobs_NestedReleaseBlobs_ExcludeRegex()
             return new BlobStorageService(
                 connectionString: "",
                 blobServiceClient ?? new Mock<BlobServiceClient>().Object,
-                new Mock<ILogger<BlobStorageService>>().Object
+                new Mock<ILogger<BlobStorageService>>().Object,
```

Comment:
Can use Mock.Of<BlobStorageService>() instead, when not needing to use the actual mock from any setups.